person.rb

```diff
@@ -1,10 +1,15 @@
-class Person
-  def initialize(id, age, name: 'Unknown', parent_permission: true)
-    @id = id
-    @id = id
+require_relative './nameable'
+require_relative './decorator'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
+
+class Person < Nameable
+  def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
### Optional
- [ ] It would be nice if we address the error as seen in the screenshot below
![Screenshot from 2023-05-24 09-12-55](https://github.com/MuhammadUsmanAslam/OOP-school-library/assets/46388113/dea82223-6b76-445e-8532-9855c6b1211f)